benchmarking/deepcell-e2e/benchmark.ipynb

```diff
@@ -37,13 +37,28 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import sys\n",
-    "if sys.prefix != sys.base_prefix:\n",
-    "  # In a venv. Just install it!\n",
-    "  ! pip install --upgrade --quiet -r ../../requirements.txt\n",
-    "else:\n",
-    "  # Not in a venv. Install to user's directory.\n",
-    "  ! pip install --user --upgrade --quiet -r ../../requirements.txt"
+    "#import sys\n",
+    "#if sys.prefix != sys.base_prefix:\n",
+    "#  # In a venv. Just install it!\n",
+    "#  ! pip install --upgrade --quiet -r ../../requirements.txt\n",
+    "#else:\n",
+    "#  # Not in a venv. Install to user's directory.\n",
+    "#  ! pip install --user --upgrade --quiet -r ../../requirements.txt"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b285eabd",
+   "metadata": {},
+   "outputs": [],
```

Comment:
Now that we've moved this to the setup notebook, let's clear it out here vs leaving it as comments.

```suggestion
```